examples/charmm/1hvn/sysinfo.dat

```diff
@@ -0,0 +1 @@
+{"membres": {"head": [], "head_upper": [], "head_lower": []}}
```

Comment:
Is this file actually needed? If so, it needs a "newline" at the end of the file, github is flagging it.